lib/zendesk_api/client.rb

```diff
@@ -142,6 +143,7 @@ def build_connection
         builder.use ZendeskAPI::Middleware::Response::Logger, config.logger if config.logger
         builder.use ZendeskAPI::Middleware::Response::ParseIsoDates
         builder.use ZendeskAPI::Middleware::Response::ParseJson
+        builder.use ZendeskAPI::Middleware::Response::SanitizeResponse
```

Comment:
should be the first so things like parsing/logging bad responses does not blow up ?